README.md

```diff
@@ -68,14 +84,69 @@ const playbackId = await Video.Assets.createPlaybackId(asset.id, {
 
 Or, if you don't have the files online already, you can ingest one via the direct uploads API.
 
+However, before that, a word of warning on using request libraries.
+
+### Request Library Usage in Node
+
+Since 2019, `request` has gone into maintenance mode. 
+
+We must use an alternate request library such as `node-fetch` [here](https://github.com/node-fetch/node-fetch/).
```

Comment:
Is there context to "We must use" that I'm unaware of? I might instead just say something like: "Instead, we use an alternate request library, `node-fetch`." Am I missing something in this?

Reply:
I think the lack of clarity might be because, "We must use," assumes the user knows we used to use request, now don't, and that we have to use an alternate request library. I'll change the language to, "Instead, we use..."